m4_sessions/automation/session_one/five_noah_eric_nicke_laszlo.md

```diff
@@ -1,3 +1,51 @@
-## Group Five's Session Plan
+# Turing Career Development: Automation
 
-M4 Student Leaders: Noah, Eric, Nick E, Laszlo
+## Session Outcomes
+
+*  [Add detail here]
+
+## Pre-Session
+
+
+### Survey Questions
+
+Survey Questions to share with instructional staff by Wednesday of the same week.
+
+1.   What value would you assign to this session (from 1-5)?
+2.   What are you taking away from this session?
+3.   What role, if any, does Turing play in taking responsibility for this issue?
+
+
+### Pre-Reading
+
+Send pre-reading out at least 24 hours before the session
+
+*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
+*   ["The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"](https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review) (_The Guardian_: October 1, 2015)
+*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+*   ["Evidence That Robots Are Winning the Race for American Jobs"](https://www.nytimes.com/2017/03/28/upshot/evidence-that-robots-are-winning-the-race-for-american-jobs.html?emc=edit_nn_20170329&nl=morning-briefing&nlid=77911801&te=1&_r=0) (_The New York Times_: 7 March 2017)
+
+### Groupings
+
+*   The group will be split equally in half
+
+
+### Logistics
+
+| Set Up | Materials Needed |
+| ------ | ---------------- |
+| Students should start the session seated in their groups. | Participants need notebooks, pens. |  |
+
+
+## Session Outline
+
+| Timing        | Facilitator Talking Points           | Participant/Facilitator Actions  |
+| ------------- | ------------- | ----- |
+| 5 min | Introduction: Welcome participants.  Introduce yourself and your role.  Review the outcomes for the session and the connection to the greater Turing mission. | Intro. |
+| 2 min | Introduce video and reflection questions to students. <br> 1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 3) What do you think it means to be a responsible citizen around this issue? <br> 4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?  <br> | Facilitor will display questions on screen. Students get out paper and pen to prepare for video. |
```

Comment:
who will pull these questions up on the screen? do you think it may be helpful for students to have these on the board possibly so they can refer to them while watching the video?